Scatternest/Menu/StartSelectionPage.cs

```diff
@@ -1,19 +1,23 @@
-﻿using MenuChanger;
+﻿using ItemChangerDataLoader;
+using MenuChanger;
 using MenuChanger.Extensions;
 using MenuChanger.MenuElements;
 using MenuChanger.MenuPanels;
+using Modding;
 using RandomizerMod.Menu;
 using RandomizerMod.RC;
+using System.Collections.Generic;
 using System.Linq;
 using static RandomizerMod.Localization;
 
 namespace Scatternest.Menu
 {
     internal class StartSelectionPage
     {
+        private static Dictionary<MenuPage, StartSelectionPage> Instances = new();
```

Comment:
I wonder if it would be cleaner to do the following:

1) Have a base class which includes the shared logic to control the appearance of the page
2) Have derived classes for the Rando and ICDL menu pages which include the non-shared static instance and methods that interface directly with Rando/ICDL
3) In the mod initialization, don't call the hook method on the ICDL menu page if ICDL is unavailable

(At any rate this ought to fix the ICDL being required issue, which I believe is caused by the compiler-generated field which holds the delegate defined in what is currently called HookICDL)

Reply:
This is not as possible as you'd think because the ICDL override actually gets invoked twice on menu initialization - once for the "Plando Plando" directory and once for the "Past Randos" directory, leading to two copies of the ICDL menu and three total elements in this map.

I added an ICDLInterop class to work around the requirement issue; tested, and it works.